src/components/pages/data/list/containers/panes.tsx

```diff
@@ -6,56 +6,78 @@ import {
 } from "@chakra-ui/icons";
 import {
   Box,
+  Button,
   Flex,
-  IconButton,
+  Icon,
   useBreakpointValue,
   useDisclosure,
   useMediaQuery
 } from "@chakra-ui/react";
+import FilterIcon from "@icons/filter";
+import ListIcon from "@icons/list";
 import React, { useEffect } from "react";
 
-const ToggleButton = ({ isOpen, onToggle, top }) => {
+const ToggleButton = ({ onToggle, top, title, isMobile, mobileTop, isOpen }) => {
   const styles = useBreakpointValue({
     base: {
-      top: "100%",
-      left: top,
-      borderTop: 0,
-      borderTopRadius: 0,
-      minW: "2.5rem",
-      h: "auto",
-      icon: isOpen ? <ChevronUpIcon /> : <ChevronDownIcon />
+      top: isOpen ? "100%" : 4,
+      left: mobileTop,
+      pr: 24,
+      height: "2.5rem",
+      width: "2.5rem",
+      icon: title === "Filters" ? <Icon as={FilterIcon} /> : <Icon as={ListIcon} />
     },
     sm: {
       top,
       left: "100%",
       borderLeft: 0,
       borderStartRadius: 0,
-      minH: "2.5rem",
-      icon: isOpen ? <ChevronLeftIcon /> : <ChevronRightIcon />
+      height: "2.4rem",
+      width: "6.2rem",
+      icon: title === "Filters" ? <Icon as={FilterIcon} /> : <Icon as={ListIcon} />
     }
   });
 
+  const chevronIcon = isOpen ? (
```

Comment:
is it possible to simplify this logic a bit? any other way we can make it more readable?